### DISCUSSION TITLE
求番，有求番的也可以在评论区问

### DISCUSSION THREAD

**Discussion by Sorasakirin** [share121/inter-knot#5242] [委托]
想要一些和心理测量者类似的悬疑番

**Comment by KELERSAMA** [comment]
某科学的超电磁炮
男子高中生的日常
向美好的世界送上祝福<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Reply by Sorasakirin** [reply]
`谢谢，全看过了`

**Comment by dahetaoa** [comment]
无职转生
紫罗兰永恒花园
中二病也要谈恋爱
白圣女与黑牧师（萌豚狂喜）
刀剑神域
夏日幽魂
re：从零开始的异世界生活
摇曳露营
赛博朋克 边缘行者
排名不分先后，先想到啥就写了啥😉
